content/install_maintain/installation/installing-external-db-and-queue-cluster.md

```diff
@@ -0,0 +1,215 @@
+---
```

Comment:
Food for thoughts, to keep it simple maybe it's better to separate External DB and External Queue to two pages.
1. It will keep the title shorter
2. makes it easier to find if you want to use external DB or external RabbitMQ